apps/prairielearn/src/lib/externalGraderSqs.ts

```diff
@@ -74,7 +74,7 @@ export class ExternalGraderSqs implements Grader {
         async () => sendJobToQueue(grading_job.id, question, config),
       ],
       (err) => {
-        fs.remove(dir);
+        void fs.remove(dir);
```

Comment:
This seems like it has the potential for an unhandled promise rejection. While we don't necessarily want to wait for the error and allow it to bubble up, we should probably make ourselves aware of if it happens. Typically we'd do something like this:

```ts
  } catch (err) {
    logger.error('Describe the error', err);
    Sentry.captureException(err);
  }
```

(Or use `.catch(...)` with the above if we're not in an async/await context)

Reply:
Added that in ed32c0fd05e656102949ea224dc4336fbd3fba0f.